R/input-file.R

```diff
@@ -91,7 +91,8 @@ fileInput <- function(inputId, label, multiple = FALSE, accept = NULL,
     id = inputId,
     name = inputId,
     type = "file",
-    style = "display: none;",
+    # Don't use "display: none;" style, which causes keyboard accessibility issue; instead use the following workaround: https://css-tricks.com/places-its-tempting-to-use-display-none-but-dont/
```

Comment:
```suggestion
    # Instead of "display: none;" style, which causes keyboard accessibility issues,
    # use workaround: https://css-tricks.com/places-its-tempting-to-use-display-none-but-dont/
```